adr/0009-add-faker-for-wanda.md

```diff
@@ -0,0 +1,24 @@
+# 9. Add Faker for Wanda
+
+Date: 2023-02-09
+
+## Status
+
+Accepted
+
+## Context
+
+A demo environment is needed to showcase the functionality of the execution server. The current implementation uses live data, which is not ideal for demonstration purposes. A Faker-based implementation will provide a safe and controlled environment for demos.
```

Comment:
I find this `context` paragraph confusing. I would start mentioning that we are talking about wanda (just saying it in the title is not enough in my opinion), and later on, that using real agents to run the evaluation is a problem for demos. 

In fact, the real decision here, and what the ADR should aim for is to tell that we decided to the "faked" code in wanda, and not in the web frontend/backend, or mocking the agents themselves, which we will probably finish doing.

After saying that, I'm struggling to decide if we even want an ADR with this "wanda" faked code now, as we might replace it by mocked agents.

Reply:
I've added a bit more context there. I've added some reasons to do thsi in wanda as part of the context. WDYT?